edns.go

```diff
@@ -673,3 +676,94 @@ func (e *EDNS0_PADDING) copy() EDNS0 {
 	copy(b, e.Padding)
 	return &EDNS0_PADDING{b}
 }
+
+const (
+	EDECodeOther uint16 = iota
```

Comment:
I would name these all `ExtendedRcodeOther` to be inline of the naming of the normal RcodeXXX

Reply:
On my reading of RFC8914, these are "info codes" and is specific to that RFC, and doesn't really seem to be an Rcode in the common DNS definition. I'm worried this naming scheme may cause confusion with the "extended Rcode" as defined in RFC6891.

Perhaps ExtendedErrorCode as a prefix might be more suitable here (or even ExtendedEcode)?

Reply:
On further thought, perhaps ExtendedErrorInfo?

Reply:
hmm good point, ExtendedRcode already exist, hmm `ErrorCode` maybe?

Reply:
I wonder if ErrorCode is a little too generic, and might be confused with something else? Ideally, I think the word "extended" should appear somewhere.

Other than that, I'm happy to go ahead with the `ErrorCode` prefix, if you think it's appropriate.